src/main/resources/config/liquibase/changelog/20201015224743_changelog.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+    <changeSet author="franke" id="20201015224743">
+        <addColumn tableName="course">
+            <column name="semester" type="VARCHAR(11)"/>
```

Comment:
The idea of limiting that to 11 is that you can define on maximum `WSYYYY/YYYY`, for example `WS2020/2021` which are 11 characters. Ping @jpbernius, @krusche.

Reply:
I would not be too strict here. Please set it to 25 in case we might have longer values in the future